src/app/components/Grid/index.jsx

```diff
@@ -21,4 +23,138 @@ export const GelPageGrid = styled(Grid)`
   ${gelMaxWidths}
 `;
 
+/* This is intented to be used on pages with single columns
+ * i.e. ArticlePage, PhotoGalleryPage and MediaAssetPage
+ */
+export const SingleColumnWrapper = styled.div`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin: 0 auto;
+    max-width: 46.4rem;
+  }
+`;
+
+const defaultColumns = {
+  group0: 6,
+  group1: 6,
+  group2: 6,
+  group3: 6,
+  group4: 6,
+  group5: 12,
+};
+
```

Comment:
I worked on Optimo articles initially and their columns were:
```
  group0: 6,
  group1: 6,
  group2: 6,
  group3: 6,
  group4: 8,
  group5: 20,
```
Then Grid items had not just a `columns` prop but a `startOffset` prop of:
```
  group0: 1,
  group1: 1,
  group2: 1,
  group3: 1,
  group4: 2,
  group5: 5,
```

e.g. https://github.com/bbc/simorgh/pull/4493/files#diff-860eef0af93e58fa85364ccd46280087R66-R80
&
https://github.com/bbc/simorgh/pull/4493/files#diff-860eef0af93e58fa85364ccd46280087R15-R26

By using `startOffset` prop on Grid items we do not need to set the max width at `46.4rem` but it can be the max width of the column at Group4. 

I'm happy to discuss this usage of startOffset in more detail, if it isn't clear from the documentation.